pom.xml

```diff
@@ -973,6 +974,12 @@
                 <artifactId>jul-to-slf4j</artifactId>
                 <version>${jul-to-slf4j.version}</version>
             </dependency>
+            <!-- rocksdb -->
+            <dependency>
+                <groupId>io.github.aliyunmq</groupId>
+                <artifactId>rocketmq-rocksdb</artifactId>
+                <version>${rocksdb.version}</version>
+            </dependency>
```

Comment:
Is this feature compatible with other platforms (Windows and macOS)? If not, should we set it as optional?

Reply:
support Windows, MacOS and Linux